packages/template-ui/src/components/ContentImage.vue

```diff
@@ -64,10 +65,12 @@ export default {
   },
   watch: {
     thumbnail() {
-      const img = new Image();
-      img.src = this.thumbnail;
-      img.onload = () => {
-        this.imageSize = img.width;
+      // Cancel previous onload events
+      this.loadingImg.onload = null;
```

Comment:
Please use explicit names: `loadingImg` should be `loadingImage`. Just 2 more characters, it shouldn't be much more typing :P Also we have smart editors nowadays.

Also, maybe starting the variable name with a verb is not a good idea. My first impression is that this is a boolean. What about `thumbnailImage`? Or just `image`, since we already have an `imageSize` variable.